### ISSUE TITLE
Use --set install_package_path to select external charts,  will get profile path error

### ISSUE THREAD

**Issue by hb-chen** [istio/istio#22368]
**Bug description**

Follow version `1.5.0` document to select external charts with `--set installPackagePath={charts-path}`. But it doesn't work.

Should  replace `installPackagePath` with `install_package_path`.
```
istioctl manifest apply \
--set install_package_path=$PWD/install/kubernetes/operator/charts
```
But this will get an error:
```
Error: open {install_package_path}/profiles/default.yaml: no such file or directory
```
Must set `profile` to a local file system path at the same time.

Use it correctly like this:
```
istioctl manifest apply \
--set profile=$PWD/install/kubernetes/operator/profiles/default.yaml \
--set install_package_path=$PWD/install/kubernetes/operator/charts
```

**Expected behavior**

`istioctl manifest apply --set install_package_path=` should use the `default` profile.

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

```
istioctl version --remote
client version: 1.5.0
control plane version: 1.5.0
data plane version: 1.5.0 (2 proxies)
```

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by elfinhe** [comment]
What's the error output of using `--set installPackagePath `?

**Comment by elfinhe** [comment]
I think the bug here -

From https://istio.io/docs/setup/install/istioctl/#install-from-external-charts,
`istioctl` assumes that users give the charts folder by installPackagePath, which does not contain the profiles/ folder. So if user uses default profile or other built-in profile, `istioctl` will look into the charts folder, e.g., .../charts/profiles/default.yaml, which doest not exist. We may change this logic let installPackagePath point the one level up, and assume charts/ and profiles/ are there.

**Comment by hb-chen** [comment]
> What's the error output of using `--set installPackagePath `?

No error, but it dos't work, https://github.com/istio/istio.io/pull/6939#issuecomment-602743312

**Comment by ostromart** [comment]
The camel vs. snake case is a red herring - in 1.5.0 I think both worked but in 1.5.1 only camel should work. The problem is indeed with the path - the proper fix touches quite a few files so it should only go into master, but I'm working on a small fix for 1.5 only. 
fyi. as a workaround the following works:
ic manifest generate --set installPackagePath=/tmp/istio-release/istio-1.5.0/install/kubernetes/operator/charts --set profile=/tmp/istio-release/istio-1.5.0/install/kubernetes/operator/profiles/default.yaml
